packages/backend/src/index.ts

```diff
@@ -105,6 +106,7 @@ function makeCreateEnv(config: Config) {
     root.child({ type: 'plugin' }),
     eventsService,
   );
+  const userInfo = new UserInfoCompat();
```

Comment:
Let's skip this and the addition to the plugin environment. We're not wiring up these services for the old backend system

Reply:
This means that `signals` will not work with the old backend system anymore as the `UserInfoService` is mandatory now?